ui/components/DataTable.tsx

```diff
@@ -59,6 +59,7 @@ export interface Props {
   dialogOpen?: boolean;
   hasCheckboxes?: boolean;
   hideSearchAndFilters?: boolean;
+  emptyMessagePlaceholder?: string | React.ReactNode;
```

Comment:
```suggestion
  emptyMessagePlaceholder?: React.ReactNode;
```

`string` seems to already be in there down in `ReactText`

```ts
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    /**
     * @deprecated Use either `ReactNode[]` if you need an array or `Iterable<ReactNode>` if its passed to a host component.
     */
    interface ReactNodeArray extends ReadonlyArray<ReactNode> {}
    type ReactFragment = {} | Iterable<ReactNode>;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```